tools/ui-components/src/button/button.tsx

```diff
@@ -46,7 +46,11 @@ const computeClassNames = ({
         'bg-default-background-danger',
         'text-default-foreground-danger',
         ...(disabled
-          ? []
+          ? [
+            'active:before:bg-default-background-quaternary',
+            'active:before:opacity-0',
+            'focus:ring-0'
```

Comment:
Instead of overridding the background of the pseudo element, we can use `hidden` instead. It will set `display: none` to the pseudo element and remove the overlay entirely.

Also, we want disabled button to be focusable, so we should keep the focus ring to help let the user know where their focus is on.

```suggestion
            'active:before:hidden',
```